src/scripts/components/surah/Line.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable consistent-return */
+
+import React from 'react';
+import * as AudioplayerActions from 'actions/AudioplayerActions';
+import debug from 'utils/Debug';
+import $ from 'jquery';
+
+class Line extends React.Component {
```

Comment:
'class' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).